build.gradle

```diff
@@ -434,6 +439,13 @@ subprojects { subproj ->
 			replaceRegex 'Empty line between last method and class closure', /\n([\s]+)}\n}\n$/, '\n$1}\n\n}\n'
 			replaceRegex 'Remove line breaks between consecutive closing parentheses', /\)\n[\s]+\)\n/, '))\n'
 		}
+
+		kotlin {
+			ktlint("0.9.0")
```

Comment:
[ktlint](https://ktlint.github.io/) is the only kotlin linter that exists at present. Its pretty well fleshed out.

If you want to have it use tabs instead of spaces for indentation it can be configured using an `.editorconfig` file in the root directory of the project.